app/javascript/mastodon/features/terms_of_service/index.jsx

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { injectIntl } from 'react-intl';
+import Column from 'mastodon/components/column';
+
+export default @injectIntl
+class PrivacyPolicy extends React.PureComponent {
+
+  static propTypes = {
+    intl: PropTypes.object,
+    multiColumn: PropTypes.bool,
+  };
+
+  state = {
```

Comment:
I think we should remove this unused `state` object. Same with `intl` propType above. They're holdovers from the Privacy Policy page that this page mimics, but we are hard-coding the date and content.